Quiz.md

```diff
@@ -136,6 +136,24 @@ __R: A diferença está relacionada a uma "exigência" da linguagem na declaraç
  |14 | C# |
  |15 | Wolfram Language |
  
+ _RESPOSTA:_
+ | Tipo | Linguagem |
+ | --- | --- |
+ | A,F,H | C |
+ | A,D,F,H | C++ |
+ | A,H | COBOL |
+ | A,F,H | Fortran |
+ | A,C,D,F,H | Java |
+ | B,E,I | Lisp |
+ | B,I | SQL |
+ | A,C,D,I | Perl |
+ | A,C,D,I | Python |
+ | A,D,H | VB |
+ | A,C,D | Lua |
+ | B | HTML |
+ | A,C,D | PHP |
+ | A,C,D | C# |
+ | A,C,D | Wolfram Language |
```

Comment:
It does not look correctly formatted to me :thinking: 

![image](https://user-images.githubusercontent.com/15305591/66173182-6984cc80-e625-11e9-80ad-c3d539cbcb87.png)
(Screenshot from https://github.com/danielscarvalho/FTT-Compilers-Quiz/blob/52fb46e0a09fdd002bd28664428fee187d67698e/Quiz.md)